src/Catalog/Extensions/DateTimeExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace System
+{
+    public static class DateTimeExtensions
+    {
+        public static DateTime ForceUtc(this DateTime date)
```

Comment:
UnitTest?

Reply:
I just moved this code from [here](https://github.com/NuGet/NuGet.Services.Metadata/pull/524/files/f358a0d4e8c8a844f32097e9499d8bd70090ce16#diff-ef9bbac124ec39795cf902ad6b03bfc2L161). As no tests existed for it, I'll create them.

Reply:
Fixed in https://github.com/NuGet/NuGet.Services.Metadata/pull/524/commits/b0ab67f19470e9ad3e6e9b6c9e6cd85817e18bcc